clang/lib/Lex/Preprocessor.cpp

```diff
@@ -234,14 +235,21 @@ void Preprocessor::FinalizeForModelFile() {
 }
 
 void Preprocessor::DumpToken(const Token &Tok, bool DumpFlags) const {
-  llvm::errs() << tok::getTokenName(Tok.getKind());
+  llvm::errs() << llvm::formatv("{0,-16} ", tok::getTokenName(Tok.getKind()));
 
-  if (!Tok.isAnnotation())
-    llvm::errs() << " '" << getSpelling(Tok) << "'";
+  std::string Spelling;
+  if (!Tok.isAnnotation()) {
+    Spelling = "'" + getSpelling(Tok) + "'";
+  }
+
+  llvm::errs() << llvm::formatv("{0,-32} ", Spelling);
```

Comment:
Should this line be included in the `!Tok.isAnnotation()` block? Otherwise, we're intentionally printing an empty string?